jobs/integration/validation.py

```diff
@@ -964,9 +964,9 @@ async def test_kubelet_extra_config(model, tools):
 
     # wait for and validate new maxPods value
     click.echo("waiting for nodes to show new pod capacity")
-    master_unit = model.applications["kubernetes-master"].units[0]
+    master_unit = model.applications["kubernetes-control-plane"].units[0]
     while True:
-        cmd = "/snap/bin/kubectl --kubeconfig /root/.kube/config -o yaml get node"
+        cmd = "/snap/bin/kubectl --kubeconfig /root/.kube/config -o yaml get node -l 'juju-application=kubernetes-worker'"
```

Comment:
is this so you ignore the control-plane?

Reply:
Correct; the test sets config on `k8s-worker` explicitly, so the verify section should only be looking at those nodes.